modules/social_features/social_group/modules/social_group_request/assets/css/social_group_popup.scss

```diff
@@ -0,0 +1,93 @@
+.social_group-popup {
```

Comment:
Isn't this something better suited in the `socialblue` theme or in `socialbase`? We'll probably need this in other cases as well, right?

cc @Kingdutch

Reply:
This CSS looks like it's doing quite a few changes to how a pop-up looks. What is the reason we're doing this as a one-off here?

Is this something we need for all pop-ups? Then we should just change the component itself.

Reply:
@agami4 , could you take a look on these questions, please

Reply:
@Kingdutch, should we move all styles for the popup to the Distro?

Reply:
@agami4 It depends. What are the changes that we are making?

Is the pop-up we have here different from others or should these changes be made for all pop-ups in Open Social? Would we need these changes if we were using e.g. the Bartik theme?

Reply:
I think makes sense to ask Xinyu about it because his design of popup has a different style than default popup. Maybe I add new styles for the all popup in the Distro.
@xinyumaGG, can you check comments above and add a feedback about popup

Reply:
FYI: we also need a better styled pop-up for the Request Event feature in #1782

Reply:
(FYI - Considering this non blocking for merging this as we also need it for request Event)

Reply:
@agami4 I discussed with Xinyu and he'd prefer all pop-ups to have the improved styling. So we can amend the pop-up styling in Socialbase/Socialblue instead of making a special-case for the group pop-up here.

Reply:
Then it makes sense to add new general styles for the popup in the Distro

Reply:
@ronaldtebrake then perhaps we can remove the styles from this PR if we separate it separately? : )

@bramtenhove You were like 60 seconds before :P 

@agami4 👍

Reply:
@Kingdutch great idea and news) When Xinyu finishes design I'll start implementing it

Reply:
@agami4 I mean the improvements that were made in this story with respect to what's currently in Open Social. So basically the changes that you made in this PR.